def main():
    try:
        other()
    except Exception as e:
        pass

def other():
    pass
